src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,22 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        // create three balls using class Lottery
+        // and print information about them in console
+
+        Ball testball = new Ball();
+        testball.setColor(ColorSupplier.getRandomColor());
+        testball.setNumber(Lotery.getRandomBall());
+        System.out.println(testball.toString());
+
+        Ball testball1 = new Ball();
+        testball1.setColor(ColorSupplier.getRandomColor());
+        testball1.setNumber(Lotery.getRandomBall());
+        System.out.println(testball1.toString());
+
+        Ball testball2 = new Ball();
+        testball2.setColor(ColorSupplier.getRandomColor());
+        testball2.setNumber(Lotery.getRandomBall());
+        System.out.println(testball2.toString());
```

Comment:
you should use Loterry class here and the method getRandomBall() to get an object of the Ball